software/ocular.yml

```diff
@@ -0,0 +1,13 @@
+name: "Ocular"
+website_url: "https://github.com/simonwep/ocular"
+source_code_url: "https://github.com/simonwep/ocular"
+description: "Simplistic and straight-forward budgeting app to track your budget across months and years."
+licenses:
+  - MIT
+platforms:
+  - Javascript
+  - Go
```

Comment:
```suggestion

```
Considering only installation method is docker, this would be good.

Reply:
Sure thing, thank you for clarifying! :)